src/prime_server.cpp

```diff
@@ -472,7 +473,7 @@ namespace prime_server {
 
     //either we just got more or we need to check the backlog
     if((force_check || messages.size()) && interrupts.find(job) != interrupts.cend())
-      throw interrupt_t(job << 32);
+      throw interrupt_t(job & 0xFFFFFFFF);
```

Comment:
Since `interrupt_t` is just an exception wrapper which builds a string, would it be possible to just have `interrupt_t`'s constructor take a `uint64_t` instead?